templates/macros/translator.html

```diff
@@ -0,0 +1,9 @@
+{%- macro translator(text, bag_name, language, fallback='de') %}
```

Comment:
explain the translator

Reply:
bereits oben etwas erklärt. einfach zu nutzen mit translator(<feld aus databag>, <databag name>, <sprache in der das Feld ausgegeben werden soll>, <ggfs andere fallback language als deutsch wenn nicht vorhanden>)